cfn-resources/network-container/cmd/resource/resource.go

```diff
@@ -206,9 +214,16 @@ func Delete(req handler.Request, prevModel *Model, currentModel *Model) (handler
 	projectID := *currentModel.ProjectId
 	containerID := *currentModel.Id
 
-	response, err := deleteContainer(client, projectID, containerID)
-	if err != nil {
-		return progressevents.GetFailedEventByResponse(fmt.Sprintf("Error getting resource : %s", err.Error()),
+	if response, err := client.Containers.Delete(context.Background(), projectID, containerID); err != nil {
+		if response.StatusCode == 409 {
+			return handler.ProgressEvent{
+				OperationStatus:  handler.Failed,
+				Message:          fmt.Sprintf("Please, make sure to delete the network peering and the atlas cluster before deleting the container: %s", err.Error()),
```

Comment:
Issue Delete Operation: you cannot delete a network container if the associated network peering is not deleted and the atlas cluster in the same region is not deleted